src/js/routes/Guide/UnknownTwitterAccount.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component, PropTypes } from "react";
+import Image from "../../components/Image";
+import { numberWithCommas, removeTwitterNameFromDescription } from "../../utils/textFormat";
+
+export default class UnknownTwitterAccount extends Component {
+  static propTypes = {
+    twitter_handle: PropTypes.string,
+    twitter_description: PropTypes.string,
+    twitter_followers_count: PropTypes.number,
+    twitter_photo_url: PropTypes.string,
+    twitter_user_website: PropTypes.string,
+    twitter_name: PropTypes.string
+  };
+
+  constructor (props) {
```

Comment:
This method does nothing. It is already inherited from Component. Should remove.